source/systems/pan/index.rst

```diff
@@ -16,12 +16,608 @@ Intel processors, from Sandy Bridge to Coffee Lake, each has
+
+
+/ptmp
+-----
+
+The /ptmp filesystems are used to stage data for FRE post-processing. /ptmp is
+not backed up and is mounted read-only by the GFDL workstations.
+
+/net, /net2, /net3
+------------------
+
+/net, /net2, and /net3 are GFDL workstation filesystems of 100 GB or more. /net
+is backed up, but /net2 and /net3 are not. /net, /net2, and /net3 are mounted
+read/write on the workstations, and read-only on the analysis hosts.
+
+In an analysis login session, use gcp to transfer files to/from the /net,
+/net2, or /net3 filesystems. For example:
+
+.. code-block:: shell
+
+   module load gcp
+   gcp /archive/USER/testfile gfdl:/net2/USER/testfile
+   gcp gfdl:/net2/USER/testfile /archive/USER/testfile
+
+/net, /net2, and /net3 are not mounted on the post-processing nodes, but they
+can be accessed via gcp.
+
+Batch Software
+==============
+
+The NOAA/RDHPCS systems in Princeton, Boulder, and Fairmont, and the DOE gaea
+system, now use the Slurm batch system.
+
+Slurm is an open-source batch system developed by DOE since 2003. It is now in
+wide use at DOE and other supercomputer sites. Slurm now includes backfill
+scheduling and accounting. Commercial support is available from SchedMD LLC.
+Slurm replaces the Moab batch system used since 2010. Tables below show
+corresponding Slurm and Moab commands and options.
+
+Since the conversion to Slurm, users of the FMS Runtime Environment (FRE) must
+use fre/bronx-15 or later on gaea and PP/AN.
+
+Access
+------
+
+Slurm can be used from any PP/AN analysis host. Logins to analysis
+automatically do "module load slurm".
+
+.. code-block:: shell
+
+   Commands
+   --------
+
+      Moab                    Slurm
+      ----                    -----
+      msub jobscript          sbatch jobscript
+      mjobctl -c jobid        scancel jobid
+
+      showq -u My.Name        squeue -l -u My.Name
+      showq -c                squeue -l --states=completed,failed
+
+      mdiag -n                sinfo -Nrl
+      msub -F "arg1 arg2" js  sbatch js arg1 arg2
+
+Local Commands
+--------------
+
+Easy-to-use local job display scripts are available on PP/AN. The '-h' or
+'--help option will display the usage messages below:
+
+.. sourcecode::
+
+   qa|qi|qr|qc [options..]
+      -u           show my jobs
+      -u user      show user's jobs
+      -j n         jobname length  (default=14)
+      -n n         username length (default=14)
+      -s           sort by node (qr) or time (qc)
+
+      qa          show all jobs
+      qi          show input queue
+      qr          show running jobs
+      qc          show completed & failed jobs
+
+   qj jobid       show job details
+   qn an|pp       show Slurm batch nodes
+
+Using these scripts, it's easy to show long jobnames. For example:
+
+.. code-block:: shell
+
+   an200> qr -j40
+
+shows all running jobs, with 40 characters of each jobname.
+
+Job Scripts
+-----------
+
+Users who do not use FRE must convert their job scripts from Moab to Slurm.
```

Comment:
Is this still relevant?  How long ago was the conversion from Moab to Slurm?  Are any HPC systems still using it?   (Probably for @underwoo / @chanwilson )

Reply:
Moab is long gone.

Reply:
Removed, thank you.